redisearch/pool.go

```diff
@@ -65,3 +66,20 @@ func (p *MultiHostPool) Get() redis.Conn {
 	return pool.Get()
 
 }
+
+func (p *MultiHostPool) Close() (err error) {
+	p.Lock()
+	defer p.Unlock()
+	for host, pool := range p.pools {
+		poolErr := pool.Close()
+		//preserve pool error if not nil but continue
```

Comment:
@itamarhaber what do you think about the following solution?

Reply:
Lovely 🔨